cloud/pkg/cloudhub/config/config.go

```diff
@@ -70,7 +70,7 @@ func InitConfigure(hub *v1alpha1.CloudHub) {
 			Config.Cert = cert
 			Config.Key = key
 		} else if !(cert == nil && key == nil) {
-			klog.Exit("Both of cert and key should be specified!")
+			panic("Both of cert and key should be specified!")
```

Comment:
Plz input an error value to panic(..) func

Reply:
Thanks, Fixed this